kit/src/components/user_image_group/mod.rs

```diff
@@ -45,6 +44,7 @@ pub fn UserImageGroup<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
                 }
             )
         } else {
+            let single_user = &cx.props.participants[0];
```

Comment:
What is this one for?

Reply:
lines 92-93 for non-group messages. the diff shows that this variable used to be define on line 33. it was moved to only be defined for non-group messages.